pubspec.yaml

```diff
@@ -10,9 +10,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
-  cupertino_icons: ^1.0.8  # Correctly placed Cupertino icons
```

Comment:
Same thing with main, firebase dependencies are missing.
Are you sure you updated ui-ux branch from develop, does your local branch also not have firebase dependencies or is it just your remote branch ? if so make sure to push to this remote branch.